ui/DesignSystem/Component/EmptyState.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
+  import Icon from "./EmptyState/Icon.svelte";
+  import { Button, Text } from "../Primitive";
+
+  export let icon = "tent";
+  export let text = "Nothing to see here";
+  export let mainCtaText = "Go back";
+  export let mainDataCy = null;
```

Comment:
I think we don't need props for the dataCy values as they will always point to the same elements, we just need to adjust the tests accordingly.

Reply:
addressed in [21e8a4b](https://github.com/radicle-dev/radicle-upstream/pull/537/commits/21e8a4bc9af3b03944c82ed1b50173ee238188a2)